fastestimator/architecture/stnet.py

```diff
@@ -0,0 +1,125 @@
+import tensorflow as tf
+
+class ReflectionPadding2D(tf.keras.layers.Layer):
```

Comment:
is this class generic enough that it would be useful to have as a separate file?

Reply:
I think this type of padding is specific to problems like image to image translation (e.g.,  cycleGAN) and style transfer.